### DISCUSSION TITLE
How to get hash of GitHub release files?

### DISCUSSION THREAD

**Discussion by snowman** [ScoopInstaller/Scoop#4748] [Help and Questions] [answered]
For example,

https://github.com/ScoopInstaller/Extras/blob/e9c5475e5c92f38847c0f7c28741cc1576e5ea06/bucket/apkeep.json

Where to get the `sha256` hash value of `2f0fedbf4243ca700be99f8a9890acf88e7b9c7f7891f25cd246ea00c3bd802a`?

**Comment by rashil2000** [comment, marked as answer]
You have to download it and compute the hash.

It's very simple.

PowerShell - `Get-FileHash file.zip`
Cmd - `certutil -hashfile file.zip SHA256`
Bash - `sha256sum file.zip`

**Comment by anderlli0053** [comment]
@snowman  You can also do it with 7Zip (you need to have it installed beforehand), just **right click on a file and select 7-Zip > CRC SHA and then select the preferred algorithm and wait for 7Zip to compute it, it will show in a new window**,  although i mostly do it with get-filehash "filename" command as rashil2000 stated.
